### PR TITLE
ci: update release names and package details

### PR DESCRIPTION
**Pull request by pjlittle**
Update release package names to use a more standard convention
that includes version. Add more details to package.json.

AB#17236

### CODE REVIEW

**Comment by mydiemho** [comment]
duplicate of #614

File: package.json [hunk 1]
```diff
@@ -100,5 +108,9 @@
         "webpack": "^4.19.1",
         "webpack-cli": "^3.1.2",
         "webpack-merge": "^4.1.5"
+    },
+    "engines": {
```
Comment (wbreza) [inline package.json]: Does this throw warning if you machine doesn't satisfy requirements?

Comment (pjlittle) [reply package.json]: for running npm commands, yep. I just put in the latest LTS release.

**Comment by pjlittle** [comment]
also, i merged the other one... this was just to work around the license/LCA check failures :man_facepalming:

**Comment by mydiemho** [comment]
> also, i merged the other one... this was just to work around the license/LCA check failures 🤦‍♂️

the cla run finished here haha

**Pull request closed without merging**